argon2/src/block.rs

```diff
@@ -151,21 +147,3 @@ impl Zeroize for Block {
         self.0.zeroize();
     }
 }
-
-#[cfg(test)]
-mod test {
-    use super::*;
-
-    #[cfg(target_arch = "x86_64")]
-    #[test]
-    fn compress_avx2() {
-        let mut lhs = Block([0; 128]);
-        lhs.0[0..7].copy_from_slice(&[0, 0, 0, 2048, 4, 2, 1]);
-        let rhs = Block([0; 128]);
-
-        let result = Block::compress_soft(&rhs, &lhs);
-        let result_avx2 = unsafe { Block::compress_avx2(&rhs, &lhs) };
-
-        assert_eq!(result.0, result_avx2.0);
-    }
-}
```

Comment:
This test feels a bit superfluous as for now the implementation is just `Block::compress` with optimizations enabled